Fix:

error: ordered comparison between pointer and zero ('const void *' and 'int')

https://bugs.scribus.net/view.php?id=16976
https://github.com/scribusproject/scribus/commit/641554ada41094f94e0ca15adad6dbdb8c52a984#diff-fd4f2a403c73388c0e3f785d2bab61b5b36a3814297b2052fc3a719cbd0b0618L161-R172
--- scribus/gtdialogs.cpp.orig	2019-03-05 16:44:41.000000000 -0600
+++ scribus/gtdialogs.cpp	2023-07-07 15:32:47.000000000 -0500
@@ -160,7 +160,7 @@
 	if (imp != "false")
 	{
 		res = imp;
-		if (importers.contains(res) > 0)
+		if (importers.contains(res))
 			ok = true;
 	}
 	
